assets/src/components/editing/editor/normalizers/tables.ts

```diff
@@ -27,16 +73,17 @@ export const normalize = (editor: Editor, node: ModelElement | FormattedText, pa
     });
 
     if (anyDiffer) {
-      node.children.forEach((row, index: number) => {
-        const children = row.children;
-        let count = children.length;
+      console.warn('Normalizing table');
```

Comment:
What do we think of having console messages during the normalization process when it makes changes? (this is a wider question than this PR)

A couple times during dev, I ran into problems where the normalizer ran over and over and there was no debug output for why that was happening.